### ISSUE TITLE
Intelligently preload the most common plants around the spawn point at the start

### ISSUE THREAD

**Issue by kb173** [boku-ilen/landscapelab#259]
Currently, plants are loaded into RAM only once they're needed by the vegetation renderer. This is optimal for startup time, but loading plant images is costly, as plants tend to be 4K images and we may need 10 of them at once for a new plant group. So loading new sections of data can be costly.

We could load a large overview land-use clip initially, call `get_most_common` and load those plants into RAM during startup. This would increase startup time by a few seconds, but loading new data when traversing the area would be a lot quicker, as many plant images are likely already loaded.

(Loading all plants into RAM at startup is not a realistic option since it'd take multiple gigabytes of RAM.)

**Comment by kb173** [comment]
Another idea: load low-resolution plants first, then exchange them for full-resolution images. However, some profiling showed that loading plant data is actually not the bottleneck right now, so not a priority.

**Comment by kb173** [comment]
After some optimizations in the vegetation system, this no longer seems to be a bottleneck at all, so we don't need to implement this as proposed.
